Frameworks/include/CALayerInternal.h

```diff
@@ -115,20 +116,16 @@ class CAPrivateInfo : public CADisplayProperties, public LLTreeNode<CAPrivateInf
 // Kicks off an update to the layer's layout and display hierarchy if needed
 - (void)_displayChanged;
 
-- (void)updateAccessibilityInfo:(const IWAccessibilityInfo*)info;
++ (CGPoint)convertPoint:(CGPoint)point fromLayer:(CALayer*)layer toLayer:(CALayer*)layer;
 
 - (void)_removeAnimation:(CAAnimation*)animation;
-- (DisplayTexture*)_getDisplayTexture;
+- (std::shared_ptr<IDisplayTexture>)_getDisplayTexture;
 
 - (CAPrivateInfo*)_priv;
 - (void)_setRootLayer:(BOOL)isRootLayer;
 
 - (void)_setZIndex:(int)zIndex;
```

Comment:
this is already heavily commented in code (see the impl in CALayer.mm)

---
In reply to: [86874169](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r86874169) [](ancestors = 86874169)